content/en/docs/architecture/images.md

```diff
@@ -0,0 +1,132 @@
+* `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6` is replaced by `ocp_builder_rhel-8-golang-1.15-openshift-4.6`
+* `registry.ci.openshift.org/ocp/4.6:base` is replaced by `base`
+
+where `ocp_builder_rhel-8-golang-1.15-openshift-4.6` and `base` are defined in `base_images`:
+
+* `base` is `ocp/4.6:base` in the [registry of the cluster in the build farm](/docs/how-tos/use-registries-in-build-farm/) where the job is running. The image is mirrored from `registry.ci.openshift.org`. Hence, the image used in the CI build is the same as `registry.ci.openshift.org/ocp/4.6:base` in `Dockerfile.rhel7`.
+* `ocp_builder_rhel-8-golang-1.15-openshift-4.6` is `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6`, also the same image as in `Dockerfile.rhel7`.
+
+We use the mirrored images in the local registry instead of pulling images from `registry.ci.openshift.org` for better efficiency.
+
+## ART Images
+
+How the ART images are built is defined in the [ocp-build-data](https://github.com/openshift/ocp-build-data) repo. For example, [cluster-etcd-operator.yml](https://github.com/openshift/ocp-build-data/blob/openshift-4.6/images/cluster-etcd-operator.yml) in the `images` folder at branch `openshift-4.6` defines the build of the ART image for `cluster-etcd-operator`. 
+
+
+```yaml
+content:
+  source:
+    dockerfile: Dockerfile.rhel7
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/cluster-etcd-operator.git
+from:
+  builder:
+  - stream: golang
+  member: openshift-enterprise-base
+```
+
+{{< alert title="Info" color="info" >}}
+As noticed, the github organization `openshift-priv` is used. Every commits merged to any repo of `openshift` is merged to the counterpart repo of `openshift-priv` as well. The role of `openshift-priv` is out of the scope of this page.
+{{< /alert >}}
+
+The above snippet defines
+* where to get `Dockerfile` of the build: `Dockerfile.rhel7` in `release-{MAJOR}.{MINOR}` branch of repo cloned from `git@github.com:openshift-priv/cluster-etcd-operator.git`. The branch name `release-{MAJOR}.{MINOR}` is resolved by the branch name of the `cluster-etcd-operator.yml` file. In the example, it is `release-4.6`.
+* how to replace the images in `From` directives of `Dockerfile.rhel7`: `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6` is replaced by `golang` and `registry.ci.openshift.org/ocp/4.6:base` is by `openshift-enterprise-base`. 
+
+## CI Images VS. ART Images
+From the above introduction, for each component, the CI image and the ART image are built from the same `Dockerfile` (DPTP has automation to refresh the `Dockerfile` path and name in the `ci-operator`'s configuration if either of them is modified in `ocp-build-data`). They are the same image if the images in `From` directives of `Dockerfile` are replaced by the same images. Actually, it is the case. In the above example, 
```

Comment:
IMO replace this section with:

While the CI and productized image builds are configured in two separate places, a number of automatic processes exist to ensure that the container images built in both environments are identical. Automation enforces that the same build manifest (`Dockerfile`) is used to define the image in both systems. Furthermore, automation also ensures that the builder images and base images that are used in CI are identical to those used by ART when building productized container images in OSBS. Therefore, with identical container build manifests and input container images, identical builds run in CI and the productized environment. Moreover, a developer can build the same container image locally, as well. [TODO: we should doc that - they need extra pull secrets I think?]

Reply:
Updated.